data/api/v1/full_spec.yaml

```diff
@@ -10803,6 +10803,20 @@ paths:
         schema:
           format: int64
           type: integer
+      - description: Include whether or not hosts are muted and when they will be unmuted
```

Comment:
I would update to something like: "Include information about the muted status of hosts and when the mute expires"